app/src/main/java/at/tugraz/ist/swe/ColorPicker.java

```diff
@@ -32,7 +32,7 @@
     private EditText textBox_Blue;
     private EditText textBox_Hex;
 
-    private AlertDialog dlg_color;
+    public AlertDialog dlg_color;
```

Comment:
please revert this change back to private

Reply:
Done.